sigle/src/ui/Form.ts

```diff
@@ -0,0 +1,89 @@
+import { styled } from '../stitches.config';
+
+export const FormRow = styled('div', {
```

Comment:
as this is not yet part of the design system I am wondering if there should be in the UI folder or not yet as we don't know the final version.
Wdyt?

Reply:
yes that's a good point, I'll move it to `StorySettingsForm.tsx` ?

Reply:
Sounds good 👍

Reply:
Can you also create an issue so we don't forget to create the components later please?

Reply:
Sure, will do that 👍